tests/integration/backup_tests/test_backups.py

```diff
@@ -72,6 +71,7 @@ async def add_writes_to_db(ops_test: OpsTest):
     await clear_writes_action.wait()
 
 
+@pytest.mark.skip("Skipping tests until fixing backup tests are addressed (DPE-4264).")
```

Comment:
tracking it here so we don't forget https://github.com/canonical/mongodb-k8s-operator/issues/249